### PR TITLE
Adds Chinese language support

### PR DESCRIPTION
**Pull request by Akiranya**

### CODE REVIEW

File: src/main/resources/chinese.yml [hunk 1]
```diff
@@ -0,0 +1,144 @@
+name: TownyResources
+version: 0.02
+language: chinese
+author: Goosius
```
Comment (Goosius1) [inline src/main/resources/chinese.yml:4]: :construction: 
- Hi, do put your own name here :-)

**Review by Goosius1 — CHANGES_REQUESTED** [review]
1 comment

File: src/main/resources/chinese.yml (hunk 1 above)
Comment (Akiranya) [reply src/main/resources/chinese.yml]: :3 done! also with some minor text improvement

File: src/main/resources/chinese.yml [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+resource_category_rabbit_hides: '兔子皮, 兔子洞'
+resource_category_rotten_flesh: '新鲜的腐肉, 尸潮'
+resource_category_bones: '骨头, 奇怪的骨骸'
+resource_category_pork: '猪肉, '
+resource_category_mutton: '羊肉, 羊群'
+resource_category_beef: '牛肉, 牛群'
+resource_category_chickens: '鸡肉, 遗弃的养鸡场'
+resource_category_eggs: '鸡蛋, 遗弃的养鸡场'
+resource_category_ender_pearls: '末影珍珠, 折跃失败的末影人'
+
```
Comment (Goosius1) [inline src/main/resources/chinese.yml:144]: :construction: 
- Hi, just this morning, I added "slime" as a new resource category.
- For completeness of your file, could you add that translation also pls before we merge.

Comment (Goosius1) [reply src/main/resources/chinese.yml:144]: I'm eager to get the (unrelated) slime fix out. So I'll say lets merge this now with just an english translation, and we can fix later

**Review by Goosius1 — APPROVED** [review]
Approved

**Pull request merged**